docs/test.dot

```diff
@@ -0,0 +1,11 @@
+digraph D {
+
+  A [shape=diamond]
+  B [shape=box]
+  C [shape=circle]
+
+  A -> B [style=dashed]
+  A -> C [color="black:invis:black"]
```

Comment:
Would be good to have a comment indicating the color is not dynamic to the color scheme of the theme:
```suggestion
  A -> C [color="black:invis:black"] // edge using a static color with transparent fill
```
At first, I thought the static color was a bug

Reply:
Done.